src/similarity.rs

```diff
@@ -145,18 +160,18 @@ fn common_ancestors(
 }
 
 // scores: maps ancestors to corresponding IC scores
-fn _mrca_and_score(scores: &HashMap<TermID, f64>) -> (Option<TermID>, f64) {
-    let mut max_ic = 0.0;
-    let mut mrca = None;
-
-    for (ancestor, ic) in scores.iter() {
-        if *ic > max_ic {
-            max_ic = *ic;
-            mrca = Some(ancestor.clone());
-        }
-    }
-    (mrca, max_ic)
-}
+// fn _mica_and_score(scores: &HashMap<TermID, f64>) -> (Option<TermID>, f64) {
```

Comment:
delete? this is all in the git repo somewhere if we need it

Reply:
Done